### PR TITLE
Add FAQ section into README

### LINKED ISSUE #33: Outdated README
The prerequisites section does not apply to latest version any more. The app has been deprecated by web app.

### PR DESCRIPTION
**Pull request by abhinayagarwal**
Fixes #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,12 @@ Prerequisites
 -------------
 
 You need a valid subscription to Gluon CloudLink. You can get it [here](http://gluonhq.com/products/cloudlink/buy/), and 
-there is a 30-day free trial. Sign up and get a valid account on Gluon CloudLink and a link to download the Gluon CloudLink 
-Dashboard. 
+there is a 30-day free trial. Sign up and get a valid account on Gluon CloudLink.
```
Comment (johanvos) [inline README.md:10]: This is not needed if you use the WaveService

**Review by erwin1 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -50,8 +29,6 @@ Instructions for Native Image
 export GRAALVM_HOME=path-to-graalvm-directory
```
Comment (jperedadnr) [inline README.md:29]: The GraalVM links are outdated, refer to docs or Gluon releases

File: README.md [hunk 3]
```diff
@@ -50,8 +29,6 @@ Instructions for Native Image
...
 ```
 
-### Desktop
-
 * Native build the application:
 ```
 mvn client:build -pl App
```
Comment (jperedadnr) [inline README.md:34]: `s/client/gluonfx`

File: README.md [hunk 4]
```diff
@@ -61,13 +38,19 @@ mvn client:build -pl App
 mvn client:run -pl App
```
Comment (jperedadnr) [inline README.md:38]: `s/client:run/gluonfx:nativerun`

**Pull request merged**